### ISSUE TITLE
Failed to execute 'transaction' on 'IDBDatabase': The storeNames parameter was empty.

### ISSUE THREAD

**Issue by Juice10** [Polarisation/indexeddb-export-import#18]
When importing a backup into a browser that didn't have an existing database I get the following error:

```
Uncaught DOMException: Failed to execute 'transaction' on 'IDBDatabase': The storeNames parameter was empty.
```

This is due to `idbDatabase.objectStoreNames` being an empty list in
```js
  const transaction = idbDatabase.transaction(
      idbDatabase.objectStoreNames,
      'readwrite'
  );
```
Any help would be appreciated!

**Comment by Polarisation** [comment]
Thanks for reporting.

Feel free to submit a Pull Request to fix this. With the fix we should include a unit test for this case.

Otherwise I'll look at it myself when I get time.

**Comment by rwwagner90** [comment]
I'm hitting this issue as well.

**Comment by Polarisation** [comment]
Thanks @rwwagner90 for the PR

Released as v2.1.1
